File: src/apis/Weather/Weather.Domain/Entities/Measure.cs
namespace Weather.Domain;

public class Measure {
    public string sId { get; set; }
    public MeasureTs ts { get; set; }
    public double preciptation { get; set; }
    public double temperature { get; set; }
    public double pressure { get; set; }
    public string label { get; set; }
    public GeoPosition position { get; set; }

}

public class MeasureTs {
    public DateTime date { get; set; }
}

File: src/webapp/WeatherApp/src/app/models/interfaces/measure.ts


export interface MeasureResponse {
    "sId": string,
    "ts": MeasureResponseDate,
    "preciptation": number, 
    "temperature": number,
    "pressure": number,
    "label": string,
    "position" : MeasureResponsePosition
}

export interface MeasureResponseDate {
    "date": Date
}

export interface MeasureResponsePosition {
    "lat": number,
    "lon": number
}

export interface MeasureRequest {
    "InitialDate": string,
    "FinalDate": string
}